src/atomicui/molecules/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -49,16 +49,22 @@ const ConfirmationModal: FC<ConfirmationModalProps> = ({
 	confirmationCheckboxName,
 	confirmationCheckboxOnChange = () => {}
 }) => {
-	const { t } = useTranslation();
+	const { t, i18n } = useTranslation();
 	const [isConfirmationChecked, setIsConfirmationChecked] = useState(false);
 	const { isMobile } = useDeviceMediaQuery();
+	const { language } = i18n;
+	const isLongLang = ["de", "es", "fr", "it", "pt-BR"].includes(language);
```

Comment:
Nit: Shouldn't this be 'isTallLang' since it's affecting height, not length?

Reply:
resolved